ParselyTracker/Sampler.swift

```diff
@@ -124,6 +126,17 @@ class Sampler {
         }
         self.samplerTimer = Timer.scheduledTimer(withTimeInterval: SAMPLE_RATE, repeats: false) { timer in self.sample() }
     }
+    
+    private func backoff(existingTimeout: TimeInterval,
+                         totalTrackedTime: TimeInterval) -> TimeInterval
+    {
+        let offsetMatchingBaseInterval: TimeInterval = TimeInterval(35)
```

Comment:
Took me a second to remember what all of these numbers mean (35, 0.3).

Let's make those globals and/or put some comments/docstrings that explain where they came from and how they're used. 

I also think changing the name from `backoff()`, which implies this function does the action of assigning the new value as well as finding it, to something that limits the reader's expectation to "computes a value and returns it" would be helpful.

Reply:
I don't think that making these global adds anything to how self-explanatory these variables are. `offsetMatchingBaseInterval` is my attempt at a variable name that clearly explains the magic of the number `35` in this context. The problem is that `35` and `.3` here are both numbers that resulted from a bunch of tinkering in a simulator and don't have clear meaning in themselves.

I agree that `backoff` could be named a lot better here.